src/NBomber/Api/FSharp.fs

```diff
@@ -188,15 +191,19 @@ module NBomberRunner =
 
             { context with InfraConfig = Some config }
 
-    /// Specifies reporting sinks.
+    /// Sets reporting sinks.
     let withReportingSinks (reportingSinks: IReportingSink list, sendStatsInterval: TimeSpan) (context: NBomberContext) =
         { context with ReportingSinks = reportingSinks
                        SendStatsInterval = sendStatsInterval }
 
-    /// Specifies plugins.
+    /// Sets plugins.
     let withPlugins (plugins: IPlugin list) (context: NBomberContext) =
         { context with Plugins = plugins }
 
+    /// Sets application type.
+    let withApplicationType (applicationType: ApplicationType) (context: NBomberContext) =
```

Comment:
add a detailed description. what the difference btw Console and Process and what is default